web/src/components/apps/AppLicense.jsx

```diff
@@ -292,7 +292,7 @@ class AppLicense extends Component {
                 </div>}
             </div>
             <div className="LicenseDetails flex flex1 justifyContent--spaceBetween">
-              <div className="Details--wrapper flex-auto flex-column">
+              <div className="flex1 flex-column u-paddingRight--20">
```

Comment:
Removing this because `Details--wrapper` has a scroll overflow and some other properties that don't need to be here. Because it as a scrolling overflow it sometimes shows scrollbars and webkit has a bug where they become permanent and mess up the layout (see screenshot below)

![unnamed](https://user-images.githubusercontent.com/4110866/172656466-bc68a407-430f-41f0-8795-b83c927d7911.png)

This should hopefully resolve those scrollbars showing.